user-guide/modules/ROOT/pages/build-options.adoc

```diff
@@ -0,0 +1,4 @@
+= Build Options
```

Comment:
In the git history, it's best to have the file move/renamed and changed (see `git mv`) than to have a file removed and another file included.